.eslintrc.json

```diff
@@ -3,6 +3,7 @@
     "next",
     "next/core-web-vitals",
     "plugin:react/recommended",
+    "plugin:@typescript-eslint/recommended",
```

Comment:
installed it because eslint had issues with global types (that they are declared but not yet used)

!important, run `npm install`